docs/data/stashcache/run-stashcache-container.md

```diff
@@ -0,0 +1,184 @@
+Running Stashcache in a Container
+=================================
+
+The OSG operates the [StashCache data federation](../stashcache/overview/), which
```

Comment:
```suggestion
The OSG operates the [StashCache data federation](/data/stashcache/overview/), which
```

Please use [site-relative links](https://opensciencegrid.org/technology/documentation/style-guide/#links), not doc-relative